src/actinia_core/core/grass_init.py

```diff
@@ -113,7 +114,6 @@ def set_grass_environment(
 
         """
         self.env["GIS_LOCK"] = str(os.getpid())
-        self.env["HOME"] = "/tmp/"
```

Comment:
Interesting. I still wonder why it worked for the webserver and not for the worker.